logstash-core/spec/logstash/util/wrapped_synchronous_queue_spec.rb

```diff
@@ -95,25 +91,32 @@ def poll(*) shift(); end
 
         it "appends batches to the queue" do
           batch = write_client.get_new_batch
-          5.times {|i| batch.push(LogStash::Event.new({"message" => "value-#{i}"}))}
+          messages = []
+          8.times do |i|
```

Comment:
Why change this from 5 to 8? Is 8 an important number to test more than 5 would be?

If 8 is not magic, then maybe we could add randomization and pick a random size for the queue to be and a random number of events to write.